src/i18n/langs/en_appInfo.json

```diff
@@ -32,6 +32,10 @@
       "jam": {
```

Comment:
@cstenglein this file is confusing, why does this exists? :P

Reply:
Is from tests to split the language files.

Can be deleted